### PR TITLE
Implementation of the Awesome Books project using ES6

### PR DESCRIPTION
**Pull request by DejazmachMolla**
In this pull request I 

- proposed change to use ES6 modules
- changed the implementation of the Date display to use an external library using Luxon

### CODE REVIEW

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  /*
+Filter out a removed book from an array of books contianed in each list item
+- add new array of books (with one less book) to the local storage
+*/
+  remove(id) {
+    this.books = this.books.filter((book) => book.id !== id);
+    localStorage.setItem('booksLocalStorage', JSON.stringify(this.books));
+    this.removeBorderFromBooks();
+  }
+
```
Comment (m05e5) [inline modules/book.js]: - [ ] As part of the requirements, please refactor your methods and functions into arrow functions 
image

File: modules/book.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  /*
+Add a book object to the books array
+- display the book on the UI
+- set booksLocalStorage variable for preserving books array during reload
+- clear currentTitle and currentAuthor variables so that those will
+  not be showing up after a book object has already been added
+- initialize event listeners for the buttons created when adding new books
+*/
+
```
Comment (m05e5) [inline modules/book.js]: - [ optional ]Hi it's not best practice to put comments everywhere in your script. your functions should be self explanatory

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @DejazmachMolla , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 
- [ ] As part of the requirements, please refactor your methods and functions into arrow functions 
image

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GiuseppeTG — APPROVED** [review]
Hi @DejazmachMolla  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights
✅ Good commit history
✅ Nice pull request
✅ Good use of ES6

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**